apps/client/src/app/core/data/lazy-data.ts

```diff
@@ -1,4 +1,6 @@
-export interface LazyData { 
+export type LazyDataKey = keyof LazyData;
```

Comment:
I know it might seem overkill, but having a file for this type will be needed, because this file is generated on the fly by the lazy data updater script.

Reply:
I actually included this line in the generator script itself.

Reply:
Oh? Didn't see it when I checked it, my bad ^^